submissions/api.py

```diff
@@ -720,6 +720,68 @@ def set_score(submission_uuid, points_earned, points_possible):
         pass
 
 
+def score_override(student_item_dict, points_possible, override_grade):
+    """
+    Create a score override for peer assessment question.
+
+    Args:
+        student_item (dict): The dictionary representation of a student item.
+        points_possible (string): Max points for this question.
+        override grade (string): Score provided by the instructor as an override.
+
+    Returns:
+        True if successful otherwise False
+    """
+    student_item_model = _get_or_create_student_item(student_item_dict)
+    try:
+        Score.create_override_score(student_item_model, points_possible, override_grade)
+    except DatabaseError:
+        msg = (
+            u"Error occurred while creating override score for"
+            u" item {item_id} in course {course_id} for student {student_id}"
+        ).format(
+            item_id=student_item_dict['item_id'],
+            course_id=student_item_dict['course_id'],
+            student_id=student_item_dict['student_id'],
+        )
+        logger.exception(msg)
+        raise SubmissionInternalError(msg)
+    else:
+        msg = u"Score overridden for item {item_id} in course {course_id} for student {student_id}".format(
```

Comment:
While I normally rant "single-quotes FTW!", I just want to emphasize that I _do_ like double-quotes here. Following shell convention, double quotes are great for representing interpolated strings.

Carry on!